src/coreComponents/constitutive/fluid/CompressibleSinglePhaseFluid.cpp

```diff
@@ -117,9 +117,9 @@ void CompressibleSinglePhaseFluid::postProcessInput()
   // Due to the way update wrapper is currently implemented, we can only support one model type
   auto const checkModelType = [&]( ExponentApproximationType const value, auto const & attribute )
   {
-    GEOSX_THROW_IF_NE_MSG( value, ExponentApproximationType::Linear,
-                           GEOSX_FMT( "{}: invalid model type in attribute '{}' (only linear currently supported)", getFullName(), attribute ),
-                           InputError );
+    GEOSX_THROW_IF( value != ExponentApproximationType::Linear,
```

Comment:
Was the old comparison macro not working properly?

Reply:
Ah, I think it is an unintentional change. It works well. Previously I modified this fluid model to make it work for both thermal and isothermal. Now I have created two separate fluid models for thermal and isothermal, but forgot to change this comparison macro back.